lib/index.js

```diff
@@ -218,11 +225,61 @@ class Embark {
     }
 
     if (cmdPlugin) {
```

Comment:
I know the code was already like that, but I feel it would be far more readable if we did 
```
if (!cmdPlugin) {
  // logs
  process.exit()
}
// Rest of the code
```